docs/sources/guides/whats-new-in-v6-6.md

```diff
@@ -61,17 +65,6 @@ Here is how it looks in light theme:
 For the panels Gauge, Bar Gauge & Stat you can now leave the min and max settings empty. Grafana will, in that case, calculate the min & max based on
 all the data.
 
-## New time picker
-
-The time picker has gotten a major design update. Key changes:
-
-- Quickly access the absolute from & to input fields without an extra click.
-- Calendar automatically shows when from or to inputs has focus
-- A single calendar view can be used to select and show the from & to date.
-- Select recent absolute ranges
-
-{{< docs-imagebox img="/img/docs/v66/time_picker_update.png" max-width="700px" caption="New time picker" >}}
-
 ## News panel
 
 This panel supports showing RSS feeds as news items. It is used in the updated default home dashboard. Add it to
```

Comment:
Minor thing maybe to note here is that it struggles with ATOM feeds, i.e. https://www.theverge.com/rss/index.xml

Reply:
maybe we should have a note about this in the panel options as well :) will add it